src/azure-cli/azure/cli/command_modules/acs/managed_cluster_decorator.py

```diff
@@ -5651,6 +5699,19 @@ def set_up_network_profile(self, mc: ManagedCluster) -> ManagedCluster:
 
         network_dataplane = self.context.get_network_dataplane()
 
+        acns = self.models.AdvancedNetworking()
```

Comment:
don't forget to set up the acns model in the update code path.

Reply:
updated